packages/@coorpacademy-components/src/molecule/multi-filter-panel/index.tsx

```diff
@@ -58,7 +59,7 @@ const MultiFilterPanel = (props: MultiFilterPanelProps) => {
         {hasSelectedFilters ? (
           <div className={style.buttonContainer}>
             <ButtonLink
-              customStyle={{fontWeight: 'normal'}}
+              customStyle={CLEAR_ALL_BUTTON_STYLE}
               label="Clear all"
```

Comment:
```suggestion
              label=translate("filter_clear_all")
```
ajoute une nouvelle locale ici locales/en/global.json
(j'avais oublié de le faire)